### ISSUE TITLE
Performance test

### ISSUE THREAD

**Issue by bbenligiray** [api3dao/signed-api#40]
using Fargate with specific specs

**Comment by bbenligiray** [comment]
Blocked by https://github.com/api3dao/signed-api/issues/9

**Comment by Siegrift** [comment]
Use Nodary pool for the test, not the Nodary API directly.

**Comment by Siegrift** [comment]
Also blocked by https://github.com/api3dao/signed-api/issues/84

**Comment by Siegrift** [comment]
> Use Nodary pool for the test, not the Nodary API directly.

Revisiting this now, I think there was a misunderstanding when we talked about it on the call. We were talking about what data provider API to use. I suggested using Nodary for that. The Nodary pool is basically a signed API.

After https://github.com/api3dao/signed-api/pull/122 is merged I propose doing this:
1. Deploy Signed API
2. Deploy Nodary API pusher with quite heavy configuration
